main.go

```diff
@@ -830,6 +833,8 @@ func parseFlags() (config, error) {
 		"The endpoint against which to send read requests for metrics. It used as a fallback to 'query.endpoint' and 'query-range.endpoint'.")
 	flag.StringVar(&rawMetricsWriteEndpoint, "metrics.write.endpoint", "",
 		"The endpoint against which to make write requests for metrics.")
+	flag.StringVar(&rawMetricsRulesEndpoint, "metrics.rules.endpoint", "",
+		"The endpoint against which to make get requests for listing recording rules and put requests for creating/updating recording rules.")
```

Comment:
The only last nit I have is that this is no longer _just_ recording rules, right? It can be alerting rules as well?

Reply:
True! updated it